packages/react/src/utilities/decorators/withViewport.tsx

```diff
@@ -123,7 +132,13 @@ export function withViewport<TProps extends { viewport?: IViewport }, TState>(
             this._events.on(win, 'resize', this._onAsyncResize);
           }
 
-          this._updateViewport();
+          if (waitForViewport) {
```

Comment:
What is the reasoning for adding an extra _updateViewPort call?  the _onAsyncResize call _updateViewPort.

Reply:
I was just being consistent between the behaviors of `componentDidMount` and `componentDidUpdate`.